benches/check.rs

```diff
@@ -24,5 +24,26 @@ pub fn check_benchmark(c: &mut Criterion) {
     });
 }
 
-criterion_group!(benches, check_benchmark);
+pub fn compare_benchmark(c: &mut Criterion) {
+    let temp = tempdir().expect("create tempdir");
+    let path = temp.into_path();
+
+    let current_dir = env::current_dir().expect("get current dir");
+    let app = dotenv_linter::cli::new(current_dir.as_os_str());
+    let matches = app.get_matches_from(vec!["dotenv-linter", "compare", ".env", ".env.compare"]);
```

Comment:
You should pass the real path of files, e.g.:
```rust
vec!["dotenv-linter", "compare", path.join(".env").to_str().expect("path to str"), path.join(".env.compare").to_str().expect("path to str")]
```

Reply:
Just followed the pattern on how the `.env` arg was passed in the check function but will update.